cal_ratio_trainer/build/build_main.py

```diff
@@ -214,6 +216,17 @@ def my_pickle_loader(f: Path) -> pd.DataFrame:
             inplace=True,
         )
 
+        df.rename(
+            columns={
+                col: col.replace("nn_", "")
+                for col in df.columns
+                if col.startswith("nn_")
+            },
+            inplace=True,
+        )
+
```

Comment:
I've totally forgotten - why is it that we need this here and not in the loader? THis is loading a different type of file so that other common code doesn't need to run?

Does that mean we have the same renaming code in multiple places (copy/paste?). Perhaps this should be extract into a common function `normalize_column_names` and put in the column names py file?

Reply:
That's my bad. It's a relic from when I was trying to figure out where the errors were coming from. I'll remove it.